src/Dummy.hpp

```diff
@@ -1,19 +1,17 @@
 #ifndef _DUMMYPROJECT_DUMMY_HPP_
 #define _DUMMYPROJECT_DUMMY_HPP_
```

Comment:
Maybe we want to switch to ```#pragma once``` here. I like this more and this will cause in the long-run less problems.